lib/ruby_lsp/setup_bundler.rb

```diff
@@ -280,5 +281,15 @@ def correct_relative_remote_paths
 
       @custom_lockfile.write(content)
     end
+
+    # Detects if the project is a Rails app by looking if the superclass of the main class is `Rails::Application`
+    sig { returns(T::Boolean) }
+    def rails_app?
+      config = Pathname.new("config/application.rb").expand_path(Dir.pwd)
```

Comment:
```suggestion
      config = Pathname.new("config/application.rb").expand_path
```
The default value for `expand_path` is already `Dir.pwd`: https://ruby-doc.org/3.3.0/File.html#method-c-expand_path
> Relative paths are referenced from the current working directory of the process unless `dir_string` is given, in which case it will be used as the starting point

Reply:
Thanks, updated!